src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2768,6 +2768,12 @@ void ApplicationManagerImpl::UnregisterApplication(
     MessageHelper::SendStopAudioPathThru(*this);
   }
 
+#ifdef SDL_REMOTE_CONTROL
+  if (!is_resuming) {
```

Comment:
@dev-gh fixed in https://github.com/LuxoftAKutsan/sdl_core/pull/199/commits/fcc3f2c1c2e14c061c7aee64e28da99b86fa211d